### PR TITLE
Labels enger

### PR DESCRIPTION
**Pull request by ethen001**
### Issues

#75 
### Overview of PR

* Made school label into a dynamic component that uses `label` label from object receive from API call

### Technical Changes
* created new `Label` component
* added `Label` component to `View Comments`
* refactored label in `AllPost` and `Individual Post` component

### CODE REVIEW

File: src/components/Comments/Comment/Comment.js [hunk 1]
```diff
@@ -78,6 +80,7 @@ Comment.propTypes = {
     content: string.isRequired,
     username: string.isRequired,
     createdAt: string.isRequired,
+    label: string.isRequired,
```
Comment (nchen000) [inline src/components/Comments/Comment/Comment.js]: `label` is not part of `comment`. You should move line 83 outside of `comment` object.

Comment (ethen001) [reply src/components/Comments/Comment/Comment.js]: I agree

File: src/components/Posts/IndividualPost/IndividualPost.js [hunk 2]
```diff
@@ -82,7 +83,7 @@ export const IndividualPost = () => {
               {post && `${post.updatedAt.substr(0, post.updatedAt.indexOf('T'))} `}
               by
               {` ${post && post.username}`}
-              <Chip label={post.label} size="small" color="primary" className={classes.chip} />
+              <Label label={post.label} style={classes.chip} />
```
Comment (uting98) [inline src/components/Posts/IndividualPost/IndividualPost.js]: would it better if only pass label as props, and move chip styling to a new style.js inside the label folder?

File: src/components/Label/__tests__/Label.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  it('renders the label', () => {
+    const { getByText } = render(<Label label="University" />);
+    expect(getByText('University')).toBeInTheDocument();
+  });
+
+  it('does not render the label', () => {
+    const { container } = render(<Label label="" />);
+    expect(container).toBeDefined();
+    expect(container.innerHTML).toBe(undefined);
+  });
```
Comment (uting98) [inline src/components/Label/__tests__/Label.js]: Please fix this test, it is failing

File: src/components/Posts/IndividualPost/IndividualPost.js (hunk 2 above)
Comment (ethen001) [reply src/components/Posts/IndividualPost/IndividualPost.js]: That is how I had it at first, but `All Post` component needs a different stylying from `Comment`s and `IndividualPost` a global styling was conflicting

**Pull request merged**